src/StellarModels/StellarModel.jl

```diff
@@ -134,11 +134,88 @@ function StellarModel(var_names::Vector{Symbol},
                       structure_equations_original=structure_equations,
                       structure_equations=tpe_stbl_funcs,
                       remesh_split_functions=remesh_split_functions,
-                      eos=eos, opacity=opacity, network=network,
+                      eos=eos, opacity=opacity, network=network, turbulence=turbulence,
                       start_step_props=start_step_props, prv_step_props=prv_step_props, props=props,
                       opt=opt, plt=plt,
                       history_file=HDF5.File(-1,""),
                       profiles_file=HDF5.File(-1,""))
     return sm
 end
 
+"""
```

Comment:
This is deprecated

Reply:
what is deprecated?

Reply:
this whole function, there is no `ssi` anymore

Reply:
Don't get you, this is the StellarModel constructor

Reply:
Ahhh, my bad, it's what's down. Yeah, that somehow revived when I merged from main. It should die